lib/vtls/unitytls.c

```diff
@@ -111,7 +111,9 @@ static unitytls_key* unitytls_key_parse_pem_from_file(const char* filepath, cons
 static bool unitytls_parse_all_pem_in_dir(struct Curl_easy* data, const char* path, unitytls_x509list* list, unitytls_errorstate* err)
 {
   bool success = false;
-#if defined(WIN32)
+#if defined(CURL_WINDOWS_APP)
```

Comment:
Talked with @aurimasc  about those functions. FindFirstFileEx is available in UWP